src/commons/components/catalog-search-panel/catalog-search-panel.tsx

```diff
@@ -19,17 +21,17 @@ export default class CatalogSearchPanel extends Component {
       <article className="catalog-search-panel">
         <section>
           <article>
-            <svg width="4em"  viewBox="0 0 640 512" fill="currentColor">
+            <svg width="4em" viewBox="0 0 640 512" fill="currentColor">
               <path d="M537.59 226.59a96 96 0 0 0-142.9-114.39A160 160 0 0 0 96 192c0 2.7.09 5.41.19 8.09A144 144 0 0 0 144 480h368a128 128 0 0 0 25.59-253.41zM384 352c0 17.67-21.5 32-48 32s-48-14.33-48-32 21.5-32 48-32a69.89 69.89 0 0 1 16 2v-76.36l-96 37.52V384c0 17.67-21.5 32-48 32s-48-14.33-48-32 21.5-32 48-32a69.89 69.89 0 0 1 16 2V239.25A16 16 0 0 1 235.19 224l128-47.25A16 16 0 0 1 384 192z"
               />
             </svg>
             <h3>Your favorite melodies as a ring back tone!</h3>
           </article>
           <article>
-            <input onChange={(e) => this.onChange(e.target.value)} type="text"/>
```

Comment:
<input onChange={this.onChange} type="text"/>
move logic into a function onChange

Example:

onChange = (e: React.ChangeEvent<HTMLInputElement>) => {
    this.setState({
      value: e.target.value
    })
  }